### ISSUE TITLE
page.main.title is translating title

### ISSUE THREAD

**Issue by koenner01** [magento/magento2#7065]
The page.main.title block is translating the title. This can lead to unwanted results (like the product name being translated...)
### Preconditions
1. MG2.1 CE
2. PHP7.0
### Steps to reproduce

1. Create a product with the name 'Save'
2. Create a storeview that doesn't use english as locale (I'm using nl_NL as example)
3. Go to the product detail page
### Expected result
1. The page title in page.main.title should be 'Save'
### Actual result
1. The page title in page.main.title is 'Opslaan' (dutch translation for save)

I traced this back to \Magento\Theme\Block\Html\Title.
Both the getPageTitle() and getPageHeading() function are wrapping the output in the magic translation function. Because of this the product name is being translated.

```
public function getPageTitle()
{
    if (!empty($this->pageTitle)) {
        return $this->pageTitle;
    }
    return __($this->pageConfig->getTitle()->getShort());
}

public function getPageHeading()
{
    if (!empty($this->pageTitle)) {
        return __($this->pageTitle);
    }
    return __($this->pageConfig->getTitle()->getShortHeading());
}
```

I'm pretty sure the translation here can be dropped all together because you can control the page titles from either the layout xml's or through the backend on storeview values.

**Comment by sevos1984** [comment]
Thanks for reporting, internal ticket created MAGETWO-59860

**Comment by hostep** [comment]
Running into the same problem over here using Magento CE 2.1.2

A client of ours uses these abstract names for their product, like 'Time' or 'Guest', and they are getting translated in the title on the product detail view, which looks really silly.

Worked around it with a custom interceptor on the `\Magento\Theme\Block\Html\Title` class:

```php
    private $request;

    public function __construct(\Magento\Framework\App\Request\Http $request)
    {
        $this->request = $request;
    }

    public function afterGetPageHeading(\Magento\Theme\Block\Html\Title $subject, $result)
    {
        if ($this->request->getFullActionName() == 'catalog_product_view')
        {
            if (!empty($subject->getPageTitle()))
            {
                return $subject->getPageTitle();
            }
        }

        return $result;
    }
```

Would be great if this issue could get some attention :)

**Comment by magento-engcom-team** [comment]
@koenner01, thank you for your report.
We've created internal ticket(s) MAGETWO-59860 to track progress on the issue.

**Comment by adrian-martinez-interactiv4** [comment]
@koenner01 @hostep PR opened to try to fix this issue in 2.3-develop, when accepted, I will backport it to 2.2-develop and 2.1-develop

**Comment by VladimirZaets** [comment]
Hi @koenner01. Thank you for your report. 
The issue has been fixed in magento/magento2#26269 by @Bartlomiejsz in 2.4-develop branch
Related commit(s):
 - [8fe06ff1c4e362160d4cbad50dfec8b4b0551b4a](https://github.com/magento/magento2/commit/8fe06ff1c4e362160d4cbad50dfec8b4b0551b4a)

The fix will be available with the upcoming 2.4.0 release.

**Comment by hostep** [comment]
Looks like a fix for this is also coming in Magento 2.3.5 (not 100% sure)
